doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst

```diff
@@ -60,94 +60,67 @@ Download instructions are in the README file `here <https://github.com/nrfconnec
-
-.. tabs::
-
-   .. group-tab:: Windows
+The process of building a SUIT envelope (which contains the manifests) can be summarized with the following diagram:
 
-      Enter the following command to change the location where the editable manifest templates are stored:
+.. figure:: images/nrf54h20_suit_generator_workflow.png
+   :alt: Modifying manifest templates workflow
 
-      .. code-block:: console
+   Modifying manifest templates workflow
 
-         west build -d C:/ncs-lcs/work_dir/build/ -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_SUIT_ENVELOPE_EDITABLE_TEMPLATES_LOCATION="C:/my_templates"
+Manifest templates (``.jinja2``) files are rendered to YAML files.
+These YAML files are used as a representation of the output binary SUIT envelopes.
+The provided manifest templates (``.jinja2``) files contain variables (represented as components), such as memory ranges, names, and paths to binaries.
+The component values are filled out automatically by the build system during the manifest rendering.
 
-   .. group-tab:: Linux
+Variables in the provided templates, like memory ranges and paths to binaries, are filled out by the build system.
 
-      Enter the following command to change the location where the editable manifest templates are stored:
+.. _ug_suit_use_own_manifest:
 
-      .. code-block:: console
+How to use your own manifest
+----------------------------
 
-         west build -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_SUIT_ENVELOPE_EDITABLE_TEMPLATES_LOCATION="/home/my_user/my_templates"
+Changing the source of manifest templates allows you to create manifest files on your own, without relying on the templates provided by Nordic Semiconductor.
 
 The source of the manifest templates can be configured by setting the following Kconfig options:
 
 * :kconfig:option:`SB_CONFIG_SUIT_ENVELOPE_ROOT_TEMPLATE`
 
-* :kconfig:option:`CONFIG_SUIT_ENVELOPE_APP_TEMPLATE`
-
-* :kconfig:option:`CONFIG_SUIT_ENVELOPE_HCI_RPMSG_SUBIMAGE_TEMPLATE`
-
-Changing the source of manifest templates allows you to create manifest files on your own, without relying on the templates provided by Nordic Semiconductor.
-
-.. _ug_suit_use_own_manifest:
-
-How to use your own manifest
-----------------------------
-
-Let us analyze the case where you would like to:
-
-* Use your own created manifest template
+* :kconfig:option:`CONFIG_SUIT_ENVELOPE_TEMPLATE` - for each of the images (application and radio core images)
 
-* Store editable manifest templates in a custom directory
+One example is demonstrated with the following case:
 
 .. tabs::
 
     .. group-tab:: Windows
 
-        The provided manifest templates are stored in ``C:\my_default_templates``.
-        Editable manifests should be stored in ``C:\my_templates``.
+        The provided user-defined manifest templates are stored in ``C:\my_default_templates``.
```

Comment:
I'm not sure if that works in command-line parameters, but from Kconfig you can also use CMake variables to manage those paths, i.e. `$(ZEPHYR_NRF_MODULE_DIR)`.
Feel free to test&update it if that simplifies the docs (i.e. results in a common code for all OSes)